.github/workflows/v1-v2-migration.yml

```diff
@@ -0,0 +1,77 @@
+##
+## This workflow is used to help facilitate the V1 -> V2 migration. 
+## It pulls down the latest V1 export, commits the new chainstate file to a feature branch,
+## and opens a PR against the master branch.
+##
+
+name: V1 to V2 Migration
+
+env:
+  COMMIT_USER: Hiro DevOps
+  COMMIT_EMAIL: 45208873+blockstack-devops@users.noreply.github.com
+  UPDATE_BRANCH: auto/v1-v2-migration
+  CHAINSTATE_URL: https://storage.googleapis.com/blockstack-v1-migration-data/chainstate.txt
+  CHAINSTATE_METADATA_URL: https://storage.googleapis.com/storage/v1/b/blockstack-v1-migration-data/o/chainstate.txt
+  CHAINSTATE_HASH_URL: https://storage.googleapis.com/blockstack-v1-migration-data/chainstate.txt.sha256
+  CHAINSTATE_HASH_METADATA_URL: https://storage.googleapis.com/storage/v1/b/blockstack-v1-migration-data/o/chainstate.txt.sha256
+  MD5_VERIFY_SCRIPT_URL: https://gist.github.com/wileyj/93b4222576cb702053ebfcd3cf9f570b
+on:
+  repository_dispatch:
+    types:
+      - migration
+
+jobs:      
+  run:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout stacks-blockchain
+        uses: actions/checkout@v2
+        with:
+          token: ${{ secrets.GH_TOKEN }}
+          ref: master
+
+      - name: Update files
+        run: |
+          curl -s ${CHAINSTATE_URL} > testnet/stacks-node/chainstate.txt
```

Comment:
@CharlieC3 This file locations have been updated, they're under the `stx-genesis` directory now. 
E.g.: https://github.com/blockstack/stacks-blockchain/blob/feat/v1-to-v2-import/stx-genesis/chainstate.txt

Reply:
Updated, thanks!